Assignment6/app.py

```
@@ -0,0 +1,207 @@
+        # Fetch company outlook data
+        company_url = f"https://api.tiingo.com/tiingo/daily/{ticker}?token={TIINGO_API_KEY}"
+        company_response = requests.get(company_url)
+        
+        if company_response.status_code == 200:
+            company_data = company_response.json()
+        else:
+            return {'error': 'No record has been found, please enter a valid symbol.'}
+        
+        # Fetch stock summary data
+        stock_url = f"https://api.tiingo.com/iex/{ticker}?token={TIINGO_API_KEY}"
+        stock_response = requests.get(stock_url)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)